doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,84 @@ menu:
     weight: 5
 ---
 
-Materialize is a **streaming database** powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
+Materialize is a **streaming database** purpose-built for low-latency
+applications. You can use it to solve problems where data must be processed at
+speeds and scales not possible in traditional databases, but without the cost,
+complexity, and development time of most streaming engines.
+
+If you need to reduce the time it takes to refresh queries that run frequently,
+or trigger user-facing actions as soon as events happen, Materialize might be a
+good fit for you! Rather than recalculate results from scratch each time it's
+queried, or serve stale cached results, Materialize continually ingests data
+and keeps results up to date for you as **incrementally updated materialized
+views**.
+
+<br>
+
+| ![streaming_analytics.gif](https://user-images.githubusercontent.com/23521087/233637537-4c2e5133-5a93-4d9d-9800-a0d4ba842c82.gif) |
+|:---------------------------------------------------------------------------------------------------------------------------------:|
+| *As an example, you can use Materialize to keep your dashboards automatically up to date.*                                                           |
```

Comment:
I like having a visual example on this page! it might be nice to make a custom visualization for this, sort of like the ones we have on the homepage of the website?

Reply:
not required! just a thought.

Reply:
I agree!